change/@fluentui-react-checkbox-8136c5be-b097-405d-909e-59dbc69c18aa.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 39 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26430&commitId=7b7c32bcbcced7c5552f625466bd8d5e2366dac0&env=prod)

✅ There was 0 screenshots added, 26 screenshots removed, 1736 screenshots unchanged, 0 screenshots with different dimensions and 13 screenshots with visible difference.


  <details><summary><b>unknown</b> 39 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|1831817| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|1832400| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|1832370| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|1827811| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|1827814| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|1823804| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|1823810| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|1821419| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|1821435| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|1821426| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|1821432| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|1832400| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|1831618| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|0| Removed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.focused.chromium.png|0| Removed|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Display Value.focused.chromium.png|0| Removed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.focused.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|0| Removed|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>